schemes/kanagawa/kanagawa.nix

```
@@ -0,0 +1,24 @@
+{
+  slug = "Kanagawa";
```

```suggestion
  slug = "kanagawa";
```
And the slug must be all lowercase